pkg/k8sutil/k8sutil.go

```diff
@@ -183,3 +184,49 @@ func isKubeMetaKind(kind string) bool {
 func isRunModeLocal() bool {
 	return os.Getenv(ForceRunModeEnv) == string(LocalRunMode)
 }
+
+// SupportsOwnerReference checks whether a given dependent supports owner references, based on the owner.
+// This function performs following checks:
+//	 -- True: Owner is cluster-scoped.
+// 	 -- True: Both Owner and dependent are Namespaced with in same namespace.
+//	 -- False: Owner is Namespaced and Dependent is Cluster-scoped.
+//   -- False: Both Owner and dependent are Namespaced with different namespaces.
```

Comment:
Nit: the alignment on this last line doesn't line up with the previous lines. Looks like a spaces/tabs issue. 

`go fmt` says the following in it's source code:
> Gofmt formats Go programs.
> It uses tabs for indentation and blanks for alignment.
> Alignment assumes that an editor is using a fixed-width font.

Since we're aligning here (not indenting) I think spaces are the correct choice.